tubearchivist/home/src/frontend/forms.py

```diff
@@ -29,13 +32,14 @@ class CustomAuthForm(AuthenticationForm):
 class UserSettingsForm(forms.Form):
     """user configurations values"""
 
+    stylesheets = get_stylesheets()
     CHOICES = [
```

Comment:
You can simplify that with something like:
```python
class UserSettingsForm(forms.Form):
    """user configurations values"""

    STYLESHEET_CHOICES = [("", "-- change stylesheet --")]
    STYLESHEET_CHOICES.extend(
        [
            (stylesheet, os.path.splitext(stylesheet)[0].title())
            for stylesheet in get_stylesheets()
        ]
    )

    stylesheet = forms.ChoiceField(
        widget=forms.Select, choices=STYLESHEET_CHOICES, required=False
    )
    page_size = forms.IntegerField(required=False)
```
This also is more explicit what is a field and what is an additional form attribute.